### ISSUE TITLE
Test: Modified elsewhere setting indicator accessibility

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#12316]
Refs: -159088

- [ ] macOS
- [ ] linux
- [ ] windows

Complexity: 3

---

In the Settings editor, there are indicators, which are labels that show up in parentheses next to some settings such as "Modified elsewhere".

1. Open the Settings editor. Clear the search query if there is one.
2. Navigate to a "Modified elsewhere" indicator. `editor.tabSize` should have one, and another easy way to make that indicator appear is to modify a setting in the workspace only, and navigate back to the User scope.
3. If the indicator says "Modified in: Workspace", then focus should only go to the `Workspace` part of the indicator. Otherwise, the entire indicator should be outlined.
4. Once focused on an indicator, press space or enter to open the hover and focus it.
5. Alternatively, one should be able to hover over an indicator with a mouse, and then click on the hover to focus it.
6. For the Modified elsewhere indicator, Tab and Shift+Tab should tab around within the hover contents only.
7. Pressing escape or another non-tab key should close the hover.
8. Pressing enter or space on a hover link should act the same as clicking on the hover link.
9. The indicator and hover should be navigable with a screenreader.
